source/math/FloatComplexVectorList.ooc

```diff
@@ -155,7 +163,9 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 		This _fastFourierTransformHelper(input, 0, input count, buffer, 0, result, 0)
 	}
 	createFFTBuffer: static func (inputSize: Int) -> This {
-		This createDefault(This _fastFourierTransformBufferSize(inputSize))
+		result := This new(This _fastFourierTransformBufferSize(inputSize))
+		result _count = inputSize
```

Comment:
`createDefault`, which was removed here, sets `_count` to the same value as `capacity`. So you probably want this function to do this:
```ooc
count := This _fastFourierTransformBufferSize(inputSize)
result := This new(count)
result _count = count
result
```
```